lms/views/api/canvas/files.py

```diff
@@ -28,7 +28,12 @@ def list_files(self):
         :raise lms.services.CanvasAPIError: if the Canvas API request fails.
             This exception is caught and handled by an exception view.
         """
-        return self.canvas.api.list_files(self.request.matchdict["course_id"])
+        course_id = self.request.matchdict["course_id"]
+        files = self.canvas.api.list_files(course_id)
+        for file in files:
+            file["id"] = f"canvas://file/course/{course_id}/file_id/{file['id']}"
```

Comment:
I think this probably will be a safer change if we updated put this into "lmsID"/"lmsURI" or similar while keeping the current id field. Same thing on BB ones.